### DISCUSSION TITLE
Is it possible to show Upload data as Download data using V2ray+websocket on a VMess server?

### DISCUSSION THREAD

**Discussion by naheedroomy** [v2fly/v2ray-core#892] [General]
I have a V2ray server on the AWS cloud. I use it to bypass some censorship with net neutrality in my country.

i'm wondering if it's possible to show whatever I upload also as download data? 

I'm new to this whole thing so sorry if this is a bad question

**Comment by xiaokangwang** [comment]
You can do this with V2Ray's grpc api.  You can use `v2ctl api --server=127.0.0.1:8080 StatsService.GetStats` to do this. You will also need to configure your V2Ray instance to export an grpc API.

Unfortunately, the document for these functions is currently not available in English. You can try machine translate on these Chinese documentations, or wait for additional community support.

https://www.v2fly.org/config/api.html#apiobject
https://www.v2fly.org/config/stats.html#%E7%94%A8%E6%88%B7%E6%95%B0%E6%8D%AE
